### PR TITLE
add a link to ftp source

### PR DESCRIPTION
**Pull request by vaikas-google**
## Proposed Changes

- Add a link to proof of concept FTP/SFTP source.
-
-

### CODE REVIEW

File: docs/eventing/sources/README.md [hunk 1]
```diff
@@ -2,62 +2,69 @@
 This is a generated file and should not be changed manually. All changes should follow the
 procedure:
 
-1. Update the information in [`sources.yaml`](./sources.yaml).
+1. Update the information in [`sources.yaml`](sources.yaml).
```
Comment (RichieEscarez) [inline docs/eventing/sources/README.md]: i see this is only a comment and does not render in GitHub nor the website (so it wont break anything). 
But FYI, due to the Hugo build, all links should be relative and any link to files within the same folder require the " ./ " prefix. 
(i will capture all these requirements in the docs contributor guide and push that out soon)

**Comment by vaikas-google** [comment]
Awesome, thanks, we should then modify our generator script. I'll do that.

File: docs/eventing/sources/README.md (hunk 1 above)
Comment (vaikas-google) [reply docs/eventing/sources/README.md]: ok, I fixed the ReadmeTemplate and reran.

Comment (vaikas-google) [reply docs/eventing/sources/README.md]: ok, I fixed the template the generator uses.
@RichieEscarez PTAL

**Pull request merged**